gc/base/MarkingScheme.hpp

```diff
@@ -139,13 +141,7 @@ class MM_MarkingScheme : public MM_BaseVirtual
 	MMINLINE bool
 	inlineMarkObject(MM_EnvironmentBase *env, omrobjectptr_t objectPtr, bool leafType=false)
 	{
-		bool didMark = false;
```

Comment:
Actually after discussing this with a few people I believe I would like these changes reverted.  You lose debugability by having all of these on the same line.  The code is way easier to add break points and debug when things are on separate lines.  Once these two functions are revered and the changesets squashed I will accept the PR.  Thanks